rfq/manager.go

```diff
@@ -917,6 +935,34 @@ func (m *Manager) RemoveSubscriber(
 	return nil
 }
 
+// GetAssetGroupKey retrieves the group key of an asset based on its ID.
+func (m *Manager) GetAssetGroupKey(ctx context.Context,
```

Comment:
I think this could be non-exported now, since it's only used through `AssetMatchesSpecifier()` at the moment.